WordPressKit/FeatureFlagRemote.swift

```diff
@@ -9,17 +9,22 @@ open class FeatureFlagRemote: ServiceRemoteWordPressComREST {
     }
 
     open func getRemoteFeatureFlags(forDeviceId deviceId: String, callback: @escaping FeatureFlagResponseCallback) {
+        self.getRemoteFeatureFlags(params: .init(deviceId: deviceId), callback: callback)
+    }
 
+    open func getRemoteFeatureFlags(params: RemoteFeatureFlagsEndpointParams, callback: @escaping FeatureFlagResponseCallback) {
```

Comment:
The new `getRemoteFeatureFlags(params:callback:)`  is only used from within the `getRemoteFeatureFlags(forDeviceId deviceId:callback:)` function. This can simplify things a bit:
```suggestion
    open func getRemoteFeatureFlags(forDeviceId deviceId: String, callback: @escaping FeatureFlagResponseCallback) {
        let params = RemoteFeatureFlagsEndpointParams(deviceId: deviceId)
```